### PR TITLE
Remove use of deprecated modules in sc up-down test

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Resolves https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/324
Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
@hangshao0 - please review

**Comment by Mesbah-Alam** [comment]
Tests on jdk11 : 

Linux: Test_grinder/job/Grinder/6015/console - passed
AIX: /Test_grinder/job/Grinder/6017/console - passed 
Windows: /Test_grinder/job/Grinder/6016/console - passed 

Test on jdk8: 
PPC64 LE Linux: Test_grinder/job/Grinder/6020/console- passed

**Comment by Mesbah-Alam** [comment]
@pshipton - could you please merge this PR?

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_IncreaseDecrease.java [hunk 1]
```diff
@@ -178,17 +178,17 @@ public void execute(StfCoreExtension test, StfSharedClassesExtension sharedClass
 		String[] expectedMessages2 = {"Cache is 100% soft full"};
 		verifyAndPrintCache(sharedClasses, SCSoftmxTestUtil.CACHE_NAME, cacheDir, SCSoftmxTestUtil.CACHE_NAME, 1, expectedMessages2);
 				
+		// Run the third workload (jvm3) in background mode that drives a new workload.
```
Comment (pshipton) [inline openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_IncreaseDecrease.java]: This comment is added twice.

Comment (Mesbah-Alam) [reply openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_IncreaseDecrease.java]: Thanks for the catch, @pshipton . I've fixed it.

**Pull request merged**